js/lib/cast.js

```diff
@@ -0,0 +1,20 @@
+
+exports.toSanitizedSequence = function toSanitizedSequence(data, targetCtor) {
```

Comment:
What is this for?

Reply:
Finally... for nothing in this PR :) but I was thinking `copyFromChannel` and `copyToChannel` had the same API as for example https://webaudio.github.io/web-audio-api/#dom-baseaudiocontext-createiirfilter which support any kind of floating point value sequence, e.g. Array, Float32Array, Float64Array (numbers is JS are basically only `double`)
Should be useful later I imagine